src/main/scala/swiss/dasch/api/ProjectsEndpointsHandler.scala

```diff
@@ -122,10 +123,12 @@ final case class ProjectsEndpointsHandler(
             prj <- projectService.findOrCreateProject(shortcode).mapError(InternalServerError(_))
             tmpDir <-
               storageService.createTempDirectoryScoped(s"${prj.shortcode}-ingest").mapError(InternalServerError(_))
-            tmpFile = tmpDir / filename.value
-            _      <- stream.run(ZSink.fromFile(tmpFile.toFile)).mapError(InternalServerError(_))
-            asset  <- ingestService.ingestFile(tmpFile, shortcode).mapError(InternalServerError(_))
-            info   <- assetInfoService.findByAssetRef(asset.ref).someOrFailException.orDie
+            tmpFile   = tmpDir / filename.value
+            _        <- stream.run(ZSink.fromFile(tmpFile.toFile)).mapError(InternalServerError(_))
+            fileSize <- Files.size(tmpFile).map(_ == 0).mapError(InternalServerError(_))
+            _        <- ZIO.fail(InternalServerError("file size zero")).when(fileSize)
```

Comment:
**issue** Return 400 Bad Request when the file size is zero

Reply:
IMHO it would also be totally fine to die in the case of an `IOException` coming from `Files.size`. It should be translated to a 500 response in that case anyway:

`Files.size(tmpFile).orDie.filterOrFail(_ < 1)(BadRequest.invalidBody("The uploaded file is empty."))`

Reply:
Great, I didn't even have to write anything! :) (Ok, the condition is flipped and the tests now require isClientError instead.)

Please take a look.